spec/features/team/invite_a_team_member_spec.rb

```diff
@@ -64,14 +66,55 @@
       end
 
       context "with an email that already exists" do
-        let(:invited_user_email) { user.email }
+        context "when the user is confirmed" do
+          let(:invited_user_email) { user.email }
+
+          it "tells the user that the email has already been taken" do
+            expect {
+              click_on "Send invitation email"
+            }.to change { User.count }.by(0)
+            expect(InviteUseCaseSpy.invite_count).to eq(0)
+            expect(page).to have_content("Email is already associated with an account. If you can't sign in, reset your password")
+          end
+        end
 
-        it "tells the user that the email has already been taken" do
-          expect {
-            click_on "Send invitation email"
-          }.to change { User.count }.by(0)
-          expect(InviteUseCaseSpy.invite_count).to eq(0)
-          expect(page).to have_content("Email is already associated with an account. If you can't sign in, reset your password")
+        context "when user is not confirmed and has no organisation" do
+          before do
+            sign_out
+            sign_up_for_account(email: invited_user_email)
+            sign_in_user user
+            visit root_path
+            click_on "Team"
+            click_on "Invite team member"
+            fill_in "Email", with: invited_user_email
+          end
+
+          let(:invited_user_email) { 'notconfirmedyet@gov.uk' }
+          let(:invited_user) { User.find_by(email: invited_user_email) }
+
+          it "sends them an invite to the organisation's account" do
+            expect {
+              click_on "Send invitation email"
+            }.to change { User.count }.by(0)
+            expect(InviteUseCaseSpy.invite_count).to eq(1)
+            expect(invited_user.confirmed?).to eq(false)
+            expect(invited_user.organisation_id).to eq(user.organisation_id)
+          end
+        end
+
+        context "when user is not confirmed but has a organisation" do
+          let(:organisation) { create(:organisation) }
+          let(:invited_user) { create(:user, organisation: organisation, confirmed_at: nil) }
+          let(:invited_user_email) { invited_user.email }
```

Comment:
This doesn't quite hit the proper flow. Here you are creating an unconfirmed user with an organisation. The real life example is that they would be invite to an org, so they would also have all the "invite" data on the user, such as `invited_at` etc.

I don't see this being an issue for now, its just not best practice. We should aim to make the testing mimic the actual user flow.